gptel-gh.el

```diff
@@ -182,25 +199,25 @@
       (setq hex (nconc hex (list (aref hex-chars (random 16))))))
     (apply #'string hex)))
 
-(defun gptel--gh-restore (file)
-  "Restore saved object from FILE."
-  (when (file-exists-p file)
+(defun gptel--gh-restore-from-file ()
```

Comment:
Can you keep this function the same signature as before, i.e., taking a file param.
I'm working on a change to automatically fetch the model list and will use this to store the cached model list as well